content/cloud/cloud-plugin.md

```diff
@@ -10,9 +10,9 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/cloud-plugin
   - [Deploy](#deploy)
   - [Login](#login)
   - [Variables](#variables)
-    - [Set (Variables)](#set-variables)
-    - [Delete (Variables)](#delete-variables)
-    - [List (Variables)](#list-variables)
+    - [Set (Variables)](#variables-set)
```

Comment:
There's a VS Code Markdown command that generates TOCs automatically from the headings (and _sometimes_ updates them), if you want to keep the TOC text lined up with the body text.

Markdown All In One is the extension with it I think.

Reply:
Thank you!